apis/model.go

```diff
@@ -18,3 +21,13 @@ type User struct {
 	LastName  string `json:"lastName" firestore:"lastName"`
 	Email     string `json:"email" firestore:"email"`
 }
+
+//StockData model
+type StockData struct {
+	StockTicker  string    `json:"stockName" firestore:"stockName"`
```

Comment:
Can you change the name on this to **Ticker** and the json tag to **ticker** to be consistent. No need for **stock** to be pre-pended since the struct is named **StockData**.

Also, if you can just remove the firestore tag since this struct's data is currently not being stored in the firestore database.